Rules/CTF/Scripts/CTF_CoinCap.as

```diff
@@ -0,0 +1,17 @@
+// hacky
+#define SERVER_ONLY
+
+const int coin_cap = 600;
+
+void onTick(CRules@ this)
+{
+	// coin cap won't exist during the christmas holiday
+	if(this.hasTag("remove coincap")) return;
```

Comment:
This PR doesn't seem to set that tag anywhere?

Reply:
derp, fixed